src/components/Analysis/Analysis.tsx

```diff
@@ -30,14 +30,12 @@ const Analysis = () => {
         <Dashboard
           accuracy={accuracy}
           data={data}
-          frames={frames}
         />
       }
       {!isInDashboard &&
         <DetailedView
           accuracy={accuracy}
           data={data}
-          frames={frames}
```

Comment:
yeah this is because we changed the way we did this right? I think im gonna get a conflict here but it will be ez to fix

Reply:
Yes, since we're not passing down the frames anymore but getting them from the data we don't need them anymore